app/code/community/TIG/MyParcel2014/Block/Adminhtml/Sales/Order/Shipment/Create/ConsignmentOptions.php

```diff
@@ -332,6 +332,55 @@ public function getIsInsured()
         return $returnArray;
     }
 
+
+    /**
+     * @param $items
+     *
+     * @return array
+     */
+    public function getIsDigitalStamp($items)
+    {
+        $itemWeight = 0;
+        foreach ($items as $item) {
+            $qty = $item->getQty();
+            $qty = $qty == null ? $item->getData('qty_ordered') : $qty;
+            if ($item instanceof Mage_Sales_Model_Order_Shipment_Item) {
+                /** @var Mage_Sales_Model_Order_Item $item */
+                $id = $item->getProductId();
+            } else {
+                /** @var Mage_Sales_Model_Quote_Address_Item $item */
+                $id = $item->getProduct()->getId();
+            }
+            var_dump($item->getWeight());
+            $parentIds = Mage::getModel('catalog/product_type_configurable')->getParentIdsByChild($id);
+            if (empty($parentIds)) {
+                $itemWeight += $item->getWeight() * $qty;
```

Comment:
Graag op 1 plek berekenen wat het totaal gewicht is.

Reply:
Here are the calculations done in a different way, because you are already in the order. It is therefore not possible to use the function `getTotalWeight`, for example. This calculation are also use different functions than the function `getTotalWeight`, this is necessary to make everything run smoothly